tests/integration/tx/aminojson/aminojson_test.go

```diff
@@ -279,9 +282,10 @@ func TestAminoJSON_LegacyParity(t *testing.T) {
 			pulsar: &ed25519.PubKey{Key: []byte("key")},
 		},
 		"crypto/secp256k1": {
```

Comment:
This works just because of the reason above, if we start using keys.pulsar.go that has the latest changes this test will fail.

Reply:
Left a comment here too. Thanks.